src/Components/NewTodo.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import users from '../api/users';
+import './Form.scss';
+
+class NewTodo extends React.Component {
+  state = {
+    userId: '',
+    title: '',
+    completed: '',
+    id: this.props.todoId,
+    titleError: false,
+    userError: false,
+
+  }
+
+  handleChangeTitle = (event) => {
+    this.setState({
+      title: event.target.value.replace(/[^a-z\s]/gi, ''),
+      titleError: false,
+    });
+  }
+
+  handleChangeUser = (event) => {
+    this.setState({
+      userId: +event.target.value,
+      userError: false,
+    });
+  }
+
+  resetForm = () => {
+    this.setState(prev => (
+      {
+        title: '',
+        completed: '',
+        userId: '',
+        id: prev.id + 1,
+
+      }
+    ));
+  }
+
+handleStatus = (event) => {
```

Comment:
```suggestion
handleStatus = ({ target }) => {
```
делай деструктуризацию